models/gc_picai_baseline/meta.json

```diff
@@ -0,0 +1,138 @@
+        "valueType": "Probability map",
+        "label": "Transverse cancer detection map",
+        "description": "Detection map of clinically significant prostate cancer lesions in 3D, where each voxel represents a floating point in range [0,1]",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "3d fullres nnUNet",
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 1200
+      },
+      "evaluation": {
+        "vol_samples": 300
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "PI-CAI challenge baseline",
+    "version": "v2.1.1",
+    "devteam": "Diagnostic Image Analysis Group, Radboud University Medical Center, Nijmegen, The Netherlands",
+    "type": "Prediction",
+    "date": {
+      "weights": "2022-06-22",
+      "code": "2022-09-05",
+      "pub": ""
+    },
+    "cite": "",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "Apache 2.0"
+    },
+    "publications": [],
+    "github": "https://github.com/DIAGNijmegen/picai_nnunet_semi_supervised_gc_algorithm",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "Prediction of the likelihood of harboring clinically significant prostate cancer (csPCa) in prostate biparametric MRI exams.",
+      "references": [
+        {
+          "label": "PI-CAI baseline algorithm on grand-challenge",
+          "uri": "https://grand-challenge.org/algorithms/pi-cai-baseline-nnu-net-semi-supervised/"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "Patient-level diagnosis performance is evaluated using the Area Under Receiver Operating Characteristic (AUROC) metric. Lesion-level detection performance is evaluated using the Average Precision (AP) metric. Overall score used to rank each AI algorithm is the average of both task-specific metrics: Overall Ranking Score = (AP + AUROC) / 2",
+      "references": [
+        {
+          "label": "PI-CAI AI challenge details",
+          "uri": "https://pi-cai.grand-challenge.org/AI/"
+        }
+      ],
+      "tables": []
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The test sets are two private cohorts of 100 and 1000 biparametric MRI exams respectively. The first was used to tune the algorithms in a public leaderboard, the second was used to determine the top 5 AI algorithms.",
+      "references": [
+        {
+          "label": "PI-CAI data section",
+          "uri": "https://pi-cai.grand-challenge.org/DATA/"
+        }
+      ],
+      "tables": []
+    },
+    "training": {
+      "title": "Training data",
+      "text": "For the PI-CAI a publicly available training datasets of 1500 biparametric MRI exams including 328 cases from the ProstateX challenge were made available.",
```

Comment:
- How was this model trained in particular (instead of describing the general case in the context of the challenge)
- How were the 1200 training samples selected out of the mentioned 1500?

Reply:
All training details are provided here: https://github.com/DIAGNijmegen/picai_baseline/blob/main/nnunet_baseline.md
Essentially, it's a custom preprocessing step followed by the standard nnU-Net pipeline (except for changing the loss function to Cross-Entropy + Focal loss).

All 1500 cases were included, with AI-derived annotations for cases without manual annotations. This publication describes how the AI-derived annotations were obtained: https://fastmri.eu/research/bosma22a.html (except for changing the radiology report based number of PI-RADS 3+ lesions to a manually determined number of ISUP 2+ lesions).

Reply:
This section has been updated with the dataset and training details.